### PR TITLE
DEVDOCS-5793: Catalog API, update 207 response description

### PR DESCRIPTION
**Pull request by bc-andreadao**
# [DEVDOCS-5793]

## What changed?
- Update 207 response description

[DEVDOCS-5793]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5793?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/catalog/products_catalog.v3.yml [hunk 1]
```diff
@@ -1152,9 +1152,9 @@ paths:
                 '207':
         '207': 
           description: |-
-            Multi-status. The product information was updated successfully, but the inventory data failed to update. 
+            Multi-status. The product information was updated successfully, but the inventory data failed to update or saving the URL failed. 
             
-            Verify that the inventory-related updates are well-formed and correct; for example, that they donʼt result in negative stock levels. Then consider updating the inventory data again.
+            If inventory data failed to update, verify that the inventory-related updates are well-formed and correct; for example, that they donʼt result in negative stock levels. Then consider updating the inventory data again.
```
Comment (kristinapototska) [inline reference/catalog/products_catalog.v3.yml:1157]: Let's add one more line for `Url failed` 
`If url failed to update, please check the url to make sure you are using latin letters, url is 255 characters max and this url is not taken by any other entity`

File: reference/catalog/products_catalog.v3.yml [hunk 2]
```diff
@@ -803,9 +803,9 @@ paths:
                           next: string
         '207':
           description: |-
-            Multi-status. The product information was updated successfully, but the inventory data failed to update. 
+            Multi-status. The product information was updated successfully, but the inventory data failed to update or saving the URL failed.
```
Comment (kristinapototska) [inline reference/catalog/products_catalog.v3.yml:806]: Let's add this sentence here as well 
"Typically indicates that a partial failure has occured, such as when a `POST` or `PUT` request is successful, but saving one of the attributes has failed"

File: reference/catalog/products_catalog.v3.yml (hunk 1 above)
Comment (bc-andreadao) [reply reference/catalog/products_catalog.v3.yml:1159]: ✔️

File: reference/catalog/products_catalog.v3.yml (hunk 2 above)
Comment (bc-andreadao) [reply reference/catalog/products_catalog.v3.yml]: ✔️

**Review by kristinapototska — DISMISSED** [review]

**Review by bc-traciporter — APPROVED** [review]
LGTM

**Pull request merged**